config/routes.rb

```diff
@@ -27,6 +27,11 @@
                  constraints: ->(_) { FeatureFlipper.show_education_benefit_form? }
     end
 
+    resource :health_care_application, only: [:create, :index] do
+      post :create
+      get :index
```

Comment:
I included it just because without when I run `rake routes` it doesn't include a route for `GET`, and if I try to go to `http://localhost:3000/v0/health_care_application` I get a 404. Wasn't sure why that was.